vignettes/pk-analysis.Rmd

```diff
@@ -0,0 +1,91 @@
+---
```

Comment:
I have really little experience with PK-Analysis, so just covered the basics. Maybe someone can add a section about user-defined PK-parameters.

Reply:
Created an issue for this https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/384